docs/source/python/getting_involved.rst

```diff
@@ -70,6 +70,6 @@ used as foundations to build easier to use entities.
   so that in the other ``.pyx`` files they can be used to implement Python classes,
   functions and helpers.
 * PyArrow is also based on dedicated pieces of code that are written into Arrow itself.
```

Comment:
I think this sentence can also be updated. While we of course still depend on "Arrow" (meaning here the C++ library, although that should probably be made explicit), it is not how this was meant now this code lives in pyarrow itself.

Reply:
Yes, agree. Will correct.

Reply:
> Apart from Arrow C++ library, which dependence is mentioned in the previous line, PyArrow is also based on PyArrow C++, dedicated pieces of code that live in ``python/pyarrow/src/arrow/python`` directory and provide the low level ...

What do you think of the change?